### DISCUSSION TITLE
How to hide and show different layer by clicking

### DISCUSSION THREAD

**Discussion by ghost** [vega/altair#3348] [Q&A] [answered]
Take the following picture as an example (note it is not produced by altair).
<img width="881" alt="image" src="https://github.com/altair-viz/altair/assets/107285219/0dd5eb51-74d0-4b85-9975-0fedc7fcefc7">

The picture contains different lines for `SMA(c,10), SMA(C,20)` etc.   When I click the name `SMA(C,10)`, it will toggle the showing of the corresponding line.

How to achieve that with altair?

**Comment by jonmmease** [comment, marked as answer]
Hi @pinkfrog9, is this interactive legend example helpful? https://altair-viz.github.io/gallery/interactive_legend.html

**Reply by ghost** [reply]
That really helps. Except the only difference is, what I want is just to toggle the clicked line alone.  I can figure it out on my own.
